quantum_systems/quantum_dots/two_dim/two_dim_ho.py

```diff
@@ -80,30 +74,18 @@ def __init__(
         self.radius = np.linspace(0, self.radius_length, self.num_grid_points)
         self.theta = np.linspace(0, 2 * np.pi, self.num_grid_points)
 
```

Comment:
Is there a reason not to call `setup_basis()` here, as is done in the 1D case? Makes usage of this basis set unintuitively different.

Reply:
The reason is laziness :P